src/library/scala/Unit.scala

```diff
@@ -41,6 +44,9 @@ object Unit extends AnyValCompanion {
    *  method is not typesafe: it accepts any Object, but will throw
    *  an exception if the argument is not a scala.runtime.BoxedUnit.
    *
+   *  The result of unboxing the value `scala.runtime.BoxedUnit.UNIT` (or `null`) is `()`.
```

Comment:
Fold into the previous paragraph?

Reply:
That would unduly stress the templating mechanism.

Reply:
especially after it's already being exploited ;)